GoogleSignIn/Sources/GIDKeychainHandler/API/GIDKeychainHandler.h

```diff
@@ -22,10 +22,13 @@ NS_ASSUME_NONNULL_BEGIN
 
 @protocol GIDKeychainHandler <NSObject>
 
+/// Loads the AuthState object from the keychain.
 - (nullable OIDAuthState *)loadAuthState;
 
+/// Saves the AuthState object to the keychain.
```

Comment:
Same comment about "AuthState." Additionally, please add a `/// - Returns: Whatever the BOOL means`. You can decide whether or not you want to document the passed `authState` parameter since IMHO it's obvious that it is what is saved.

Reply:
Done.